jsapp/js/router/legacy.tsx

```diff
@@ -84,3 +83,8 @@ export function routerGetAssetId() {
   }
   return null;
 }
+
+export let router: any = null;
```

Comment:
Just a note: let's make sure we will change this `any` to something useful before merging

Reply:
`import type { Router } from "@remix-run/router";`